src/modules/resource_monitor.py

```diff
@@ -27,15 +30,20 @@ def _get_delay_query(self):
 
     def _get_delay(self):
         query = self._get_delay_query()
-        delay = self.psql_conn.select_single(query) or 0
-        return delay
+        value = self.loop.run_until_complete(
+            self.psql_conn.select_single(query))
+        if value is None:
+            return 0
```

Comment:
The logic should be the same as in FireJacket. Should it return -1 for None?

Reply:
I had added it